### PR TITLE
CC - Add example for route decorator in README

### LINKED ISSUE #8: Support for route decorators?
Currently, to use aiohttp-swagger3, one needs to use SwaggerDocs.add_routes():

```
    s = SwaggerDocs(app, '/docs', title="Swagger Petstore", version="1.0.0", components="components.yaml")
    s.add_routes([
        web.get("/pets/{pet_id}", get_one_pet),
    ])
```

Is it possible to use aiohttp decorator format?
```
routes = web.RouteTableDef()

@routes.get('/')
async def hello(request):
    return web.Response(text="Hello, world")

app = web.Application()
app.add_routes(routes)
web.run_app(app)
```

A example in my mind is like what aiohttp-swagger, where you can define the app the way you want, then add `setup_swagger(app)`. That style is more easily to adapt the existing code base. (https://aiohttp-swagger.readthedocs.io/en/latest/quick_start.html):
```
from aiohttp import web
from aiohttp_swagger import *


async def ping(request):
    """
    ---
    description: This end-point allow to test that service is up.
    tags:
    - Health check
    produces:
    - text/plain
    responses:
        "200":
            description: successful operation. Return "pong" text
        "405":
            description: invalid HTTP Method
    """
    return web.Response(text="pong")


app = web.Application()
app.router.add_route('GET', "/ping", ping)

setup_swagger(app)

web.run_app(app, host="127.0.0.1")
```

Is this something that can be added easily?

### PR DESCRIPTION
**Pull request by charleswhchan**
Fix #8

### CODE REVIEW

**Comment by hh-h** [comment]
First of all, thank you :) But can you move it under /examples. I don't want to overwhelm README with more then one example.

**Comment by charleswhchan** [comment]
@hh-h : moved to `/examples` as requested. Please check the PR again. Thanks.

File: examples/route_decorator/petstore.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+openapi: "3.0.0"
```
Comment (hh-h) [inline examples/route_decorator/petstore.yaml]: components file should have only components section, I'll add a check for this.

**Comment by hh-h** [comment]
added example https://github.com/hh-h/aiohttp-swagger3/commit/35be7347cd6272b4cc85520c0f4ab15125cd6dce

**Pull request closed without merging**